### ISSUE TITLE
OSError: [Errno 24] Too many open files

### ISSUE THREAD

**Issue by leejunho-com** [week-end-manufacture/incoming#41]
# Error message
```bash
=IC VIDEO PROCESS END=
=IC COPY PROCESS START=
=IC COPY PROCESS END=
=IC UNLINK PROCESS START=
Traceback (most recent call last):
  File "/opt/homebrew/bin/incoming", line 8, in <module>
  File "/opt/homebrew/Cellar/incoming/beta0.0.2/libexec/lib/python3.9/site-packages/incoming/main.py", line 216, in main
  File "/opt/homebrew/Cellar/incoming/beta0.0.2/libexec/lib/python3.9/site-packages/incoming/icpostprocessing.py", line 13, in _init
  File "/opt/homebrew/Cellar/incoming/beta0.0.2/libexec/lib/python3.9/site-packages/incoming/ic_log.py", line 32, in init_logger
  File "/opt/homebrew/Cellar/python@3.9/3.9.19/Frameworks/Python.framework/Versions/3.9/lib/python3.9/posixpath.py", line 380, in abspath
OSError: [Errno 24] Too many open files
```
after videdo process this message shows up

# code
https://github.com/week-end-manufacture/incoming/blob/5c5f67cf47898286f74298fefee096d80430a3da/incoming/ic_postprocessing.py#L15-L35

# doubt
![image](https://github.com/week-end-manufacture/incoming/assets/91662691/a2d1fd37-7440-4f6c-81a9-d79bef6640f7)
설마...
icpost... >> ic_post...

**Comment by leejunho-com** [comment]
❯ ulimit -n
256

**Comment by JoHyukJun** [comment]
yap. but i have new solution.
version upgraded. test it!

**Comment by leejunho-com** [comment]
# Error message
```bash
=IC VIDEO PROCESS END=
=IC COPY PROCESS START=
=IC COPY PROCESS END=
=IC UNLINK PROCESS START=
Traceback (most recent call last):
  File "/opt/homebrew/bin/incoming", line 8, in <module>
  File "/opt/homebrew/Cellar/incoming/beta0.0.4/libexec/lib/python3.9/site-packages/incoming/main.py", line 216, in main
  File "/opt/homebrew/Cellar/incoming/beta0.0.4/libexec/lib/python3.9/site-packages/incoming/ic_postprocessing.py", line 12, in __init__
  File "/opt/homebrew/Cellar/incoming/beta0.0.4/libexec/lib/python3.9/site-packages/incoming/ic_log.py", line 41, in __init__
  File "/opt/homebrew/Cellar/python@3.9/3.9.19/Frameworks/Python.framework/Versions/3.9/lib/python3.9/posixpath.py", line 380, in abspath
OSError: [Errno 24] Too many open files
```

# version
```bash
❯ incoming -v
!!!START ICOMING PROGRAM!!!
incoming beta0.0.4
```

# input files
```bash
`-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ S1+S2\ [\ x265-10Bit\ Ver.\ ]\ -\ TV\ +\ SP
    |-- EXTRA
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP00]\ Menu\ -\ 01\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP00]\ Menu\ -\ 02\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP00]\ Menu\ -\ 03\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP00]\ Menu\ -\ 04\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP00]\ Menu\ -\ 05\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP00]\ Menu\ -\ 06\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP01]\ NCOP\ -\ 01\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP01]\ NCOP\ -\ 02\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP02]\ NCED\ -\ 01\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP02]\ NCED\ -\ 02\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP03]\ S1\ Teaser\ PV\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP03]\ S2\ Teaser\ PV\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP04]\ S1\ Main\ PV\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP04]\ S2\ Main\ PV\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP05]\ CM\ -\ 01\ [\ Today\ Broadcasting\ ]\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP05]\ CM\ -\ 02\ [\ Before\ Broadcasting\ ]\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP05]\ CM\ -\ 03\ [\ While\ Broadcasting\ ]\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP06]\ 4min.\ Understanding\ Boku\ Yaba\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP07]\ TV-CM\ -\ 01\ [\ Before\ Broadcasting\ ]\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP07]\ TV-CM\ -\ 02\ [\ While\ Broadcasting\ ]\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP08]\ S2\ Making\ PV\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 01\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 02\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 03\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 04\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 05\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 06\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 07\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 08\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 09\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 10\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 11\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 12\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 13\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 14\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 15\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 16\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 17\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 18\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 19\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 20\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 21\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 22\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 23\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 24\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP09]\ Web\ Yokoku\ -\ 25\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP10]\ Special\ Stage\ Before\ Broadcasting\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP11]\ Special\ Stage\ Before\ Last\ Episode\ Broadcasting\ (BD\ 1920x1080\ HEVC-YUV420P10\ FLAC).mkv
    |   `-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ [SP12]\ BD\ Scans
    |       |-- [SP12]\ BD\ Scan\ -\ 01
    |       |   |-- 01.png
    |       |   |-- 02.png
    |       |   |-- 03.png
    |       |   |-- 04.png
    |       |   |-- 05.png
    |       |   |-- 06.png
    |       |   |-- 07.png
    |       |   |-- 08.png
    |       |   |-- 09.png
    |       |   |-- 10.png
    |       |   |-- 11.png
    |       |   |-- 12.png
    |       |   |-- 13.png
    |       |   |-- 14.png
    |       |   |-- 15.png
    |       |   |-- 16.png
    |       |   |-- 17.png
    |       |   |-- 18.png
    |       |   |-- 19.png
    |       |   `-- 20.png
    |       |-- [SP12]\ BD\ Scan\ -\ 02
    |       |   |-- 01.png
    |       |   |-- 02.png
    |       |   |-- 03.png
    |       |   |-- 04.png
    |       |   |-- 05.png
    |       |   |-- 06.png
    |       |   |-- 07.png
    |       |   |-- 08.png
    |       |   |-- 09.png
    |       |   |-- 10.png
    |       |   |-- 11.png
    |       |   |-- 12.png
    |       |   |-- 13.png
    |       |   |-- 14.png
    |       |   |-- 15.png
    |       |   |-- 16.png
    |       |   |-- 17.png
    |       |   |-- 18.png
    |       |   |-- 19.png
    |       |   `-- 20.png
    |       |-- [SP12]\ BD\ Scan\ -\ 03
    |       |   |-- 01.png
    |       |   |-- 02.png
    |       |   |-- 03.png
    |       |   |-- 04.png
    |       |   |-- 05.png
    |       |   |-- 06.png
    |       |   |-- 07.png
    |       |   |-- 08.png
    |       |   |-- 09.png
    |       |   |-- 10.png
    |       |   |-- 11.png
    |       |   |-- 12.png
    |       |   |-- 13.png
    |       |   |-- 14.png
    |       |   |-- 15.png
    |       |   |-- 16.png
    |       |   |-- 17.png
    |       |   |-- 18.png
    |       |   |-- 19.png
    |       |   |-- 20.png
    |       |   `-- BOX
    |       |       |-- 01.png
    |       |       |-- 02.png
    |       |       |-- 03.png
    |       |       |-- 04.png
    |       |       |-- 05.png
    |       |       `-- 06.png
    |       |-- [SP12]\ BD\ Scan\ -\ 04
    |       |   |-- 01.png
    |       |   |-- 02.png
    |       |   |-- 03.png
    |       |   |-- 04.png
    |       |   |-- 05.png
    |       |   |-- 06.png
    |       |   |-- 07.png
    |       |   |-- 08.png
    |       |   |-- 09.png
    |       |   |-- 10.png
    |       |   |-- 11.png
    |       |   |-- 12.png
    |       |   |-- 13.png
    |       |   |-- 14.png
    |       |   |-- 15.png
    |       |   |-- 16.png
    |       |   |-- 17.png
    |       |   |-- 18.png
    |       |   |-- 19.png
    |       |   |-- 20.png
    |       |   |-- 21.png
    |       |   `-- 22.png
    |       |-- [SP12]\ BD\ Scan\ -\ 05
    |       |   |-- 01.png
    |       |   |-- 02.png
    |       |   |-- 03.png
    |       |   |-- 04.png
    |       |   |-- 05.png
    |       |   |-- 06.png
    |       |   |-- 07.png
    |       |   |-- 08.png
    |       |   |-- 09.png
    |       |   |-- 10.png
    |       |   |-- 11.png
    |       |   |-- 12.png
    |       |   |-- 13.png
    |       |   |-- 14.png
    |       |   |-- 15.png
    |       |   |-- 16.png
    |       |   |-- 17.png
    |       |   |-- 18.png
    |       |   |-- 19.png
    |       |   `-- 20.png
    |       `-- [SP12]\ BD\ Scan\ -\ 06
    |           |-- 01.png
    |           |-- 02.png
    |           |-- 03.png
    |           |-- 04.png
    |           |-- 05.png
    |           |-- 06.png
    |           |-- 07.png
    |           |-- 08.png
    |           |-- 09.png
    |           |-- 10.png
    |           |-- 11.png
    |           |-- 12.png
    |           |-- 13.png
    |           |-- 14.png
    |           |-- 15.png
    |           |-- 16.png
    |           |-- 17.png
    |           |-- 18.png
    |           |-- 19.png
    |           `-- 20.png
    |-- ScreenShot
    |   |-- 01.\ BD.png
    |   |-- 02.\ MKV.png
    |   |-- 03.\ BD.png
    |   |-- 04.\ MKV.png
    |   |-- 05.\ BD.png
    |   |-- 06.\ MKV.png
    |   |-- 07.\ BD.png
    |   |-- 08.\ MKV.png
    |   |-- 09.\ BD.png
    |   |-- 10.\ MKV.png
    |   |-- 11.\ BD.png
    |   `-- 12.\ MKV.png
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 01\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLACx2).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 02\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 03\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 04\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 05\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 06\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLACx2).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 07\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 08\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 09\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 10\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 11\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 12\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLACx2).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 12.5\ [\ Twi-Yaba\ ]\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 13\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 14\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 15\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLACx2).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 16\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 17\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 18\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLACx2).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 19\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 20\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 21\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 22\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 23\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLAC).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 24\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLACx2).mkv
    |-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ -\ 25\ END\ (BD\ 1920x1080\ HEVC-YUV444P10\ FLACx2).mkv
    `-- [Moozzi2]\ Boku\ no\ Kokoro\ no\ Yabai\ Yatsu\ S1+S2\ [\ x265-10Bit\ Ver.\ ]\ -\ TV\ +\ SP.md5

12 directories, 216 files
```

**Comment by leejunho-com** [comment]
beta 0.0.5 fixed

**Comment by leejunho-com** [comment]
'''bash
=IC VIDEO PROCESS END=
=IC COPY PROCESS START=
=IC COPY PROCESS END=
=IC UNLINK PROCESS START=
Traceback (most recent call last):
  File "/opt/homebrew/bin/incoming", line 8, in <module>
  File "/opt/homebrew/Cellar/incoming/beta0.0.5/libexec/lib/python3.9/site-packages/incoming/main.py", line 218, in main
  File "/opt/homebrew/Cellar/incoming/beta0.0.5/libexec/lib/python3.9/site-packages/incoming/ic_postprocessing.py", line 12, in __init__
  File "/opt/homebrew/Cellar/incoming/beta0.0.5/libexec/lib/python3.9/site-packages/incoming/ic_log.py", line 49, in __init__
  File "/opt/homebrew/Cellar/python@3.9/3.9.19/Frameworks/Python.framework/Versions/3.9/lib/python3.9/posixpath.py", line 380, in abspath
OSError: [Errno 24] Too many open files
```

**Comment by JoHyukJun** [comment]
resolved issue; beta 0.0.6
